cloudevents/formats/cloudevents-compact.avsc

```diff
@@ -0,0 +1,79 @@
+{
+  "namespace": "io.cloudevents.v1.avro.compact",
+  "type": "record",
+  "name": "CloudEvent",
+  "version": "1.0",
+  "doc": "Avro Compact Event Format for CloudEvents",
+  "fields": [
+    {
+      "name": "id",
+      "type": "string"
+    },
+    {
+      "name": "source",
+      "type": "string"
+    },
+    {
+      "name": "type",
+      "type": "string"
+    },
+    {
+      "name": "datacontenttype",
+      "type": [
+        "null",
+        "string"
+      ],
+      "default": null
+    },
+    {
+      "name": "dataschema",
+      "type": [
+        "null",
+        "string"
+      ],
+      "default": null
+    },
+    {
+      "name": "subject",
+      "type": [
+        "null",
+        "string"
+      ],
+      "default": null
+    },
+    {
+      "name": "time",
+      "type": [
+        "null",
+        {
+          "type": "long",
+          "logicalType": "timestamp-micros"
+        }
+      ],
+      "default": null
+    },
+    {
+      "name": "attributes",
+      "type": {
+        "type": "map",
+        "values": [
+          "null",
+          "boolean",
+          "int",
+          "long",
```

Comment:
I believe this should be represented as follows to clarify that we're talking about 'timestamp' types..

```
...
"boolean",
"int",
{ 
  "type": "long",
  "logicalType" : "timestamp-micros"
}
"string",
....
```

Reply:
but then you could not have any `long`

Reply:
That's true .. but "long" is not a supported CloudEvent attribute primitive  type ...

Reply:
ah - did not know that